maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java

```diff
@@ -67,6 +79,7 @@
 
     // TODO: Create and pass in suitable parameters to Aprox to create the
     //       proxy repository.
+    @Deprecated
```

Comment:
I think the buildContentSetId was a holdover from before we decided to do the common component pool. I would have been necessary for chained builds before that. Now, with CCP I think we can use the untested builds artifacts collection as a rough equivalent to the chain build set, since promotion into this untested level should be a prerequisite for any dependency projects to build.